class-activity-2.Rmd

```diff
@@ -64,89 +63,160 @@ D4 <- filter(D1, DBN == "31R075"|DBN == "01M015"| DBN == "01M345")
 D4 <- droplevels(D4)
 boxplot(D4$total_enrollment ~ D4$DBN)
 ```
-#Pairs
+  
+### Pairs  
 ```{r}
 #Use matrix notation to select columns 5,6, 21, 22, 23, 24
 D5 <- D2[,c(5,6, 21:24)]
 #Draw a matrix of plots for every combination of variables
 pairs(D5)
 ```
-# Exercise
+  
+## Exercise  
+##### 1. Create a simulated data set containing 100 students, each with a score from 1-100 representing performance in an educational game. The scores should tend to cluster around 75. Also, each student should be given a classification that reflects one of four interest groups: sport, music, nature, literature.
 
-1. Create a simulated data set containing 100 students, each with a score from 1-100 representing performance in an educational game. The scores should tend to cluster around 75. Also, each student should be given a classification that reflects one of four interest groups: sport, music, nature, literature.
-
-```{r}
+```{r data simulation}
+set.seed(123)  
 #rnorm(100, 75, 15) creates a random sample with a mean of 75 and standard deviation of 20
+StudentScores <- rnorm(100, 75, 20)
+StudentScores <- runif(100, min = 1, max = 100)
 #pmax sets a maximum value, pmin sets a minimum value
+StudentScores <- pmax(1, StudentScores)
+StudentScores <- pmin(100, StudentScores)
```

Comment:
There is an another way of setting maximum and minimum values with a single line of code. `round(pmax(1, pmin(100, rnorm(100, 75, 20))))`